lib/tasks/bundlers/generateManifestBundle.js

```diff
@@ -11,25 +12,29 @@ const BUNDLE_NAME = "manifest-bundle.zip";
  * @param {Object} parameters Parameters
  * @param {module:@ui5/fs.DuplexCollection} parameters.workspace DuplexCollection to read and write files
  * @param {Object} parameters.options Options
- * @param {string} parameters.options.namespace Namespace of the application
+ * @param {string} parameters.options.projectName Project name
+ * @param {string} [parameters.options.namespace] Namespace of the application (can be omitted for libraries)
```

Comment:
Because it is needed and I mixed my brain with the cachebuster info and the manifest bundle creator... :-/ Thanks for your exact review and your patience with me... :-)